DigitalLearningSolutions.Web/Views/Frameworks/Developer/_SignpostingCard.cshtml

```diff
@@ -44,15 +44,15 @@
             Minimum result match
```

Comment:
Can we change this markup to use the same approach we use in the set signposting parameters summary screen for radio button questions? It looks odd if the user only chooses one response.